message.mk

```diff
@@ -85,3 +85,5 @@ MSG_PYTHON_MISSING = $(WARN_COLOR)WARNING:$(NO_COLOR)\n \
 	Python 3 is not installed. It will be required by a future version\n\
 	of qmk_firmware.\n\n\
 	Please run $(BOLD)util/qmk_install.sh$(NO_COLOR) to install all the dependencies QMK requires.\n\n
+MSG_FLASH_BOOTLOADER = $(WARN_COLOR)WARNING:$(NO_COLOR) This board's bootloader is not supported by the \":flash\" target at this time.\n\n
```

Comment:
Is there a specific msg for keyboards that are missing the bootloader value all together?

Reply:
No, If the bootloader isn't supported, or has none set, it gives this message.

Reply:
I'd suggest updating it to something like "bootloader unsupported or unspecified"

Reply:
You mean something like: 

```suggestion
MSG_FLASH_BOOTLOADER = $(WARN_COLOR)WARNING:$(NO_COLOR) This board's bootloader is not specified or is not supported by the \":flash\" target at this time.\n\n
```

Reply:
Yup, that looks great